smdebug/tensorflow/keras.py

```diff
@@ -69,8 +64,7 @@ def _is_not_supported(self):
             if tf.executing_eagerly() or (
                 hasattr(self.model, "run_eagerly") and self.model.run_eagerly
             ):
-                self.logger.info("Disabling SMDebug as it does not support eager mode")
-                self._hook_supported = False
+                pass
```

Comment:
why removed above 2 line and just silently pass?

Reply:
we do have support for eager mode. based on tests for 1.x (https://github.com/awslabs/sagemaker-debugger/blob/d446b26ca11f3ff655f24441af749a26eab560d1/tests/tensorflow/keras/test_keras.py#L142), eager mode did work before as well. not sure why the hook was disabled.

Reply:
please double check with  rahul & Jared , if there is any reason why it was done like that before.

Reply:
@Vikas-kum for now, i could keep this log and hook_supported=False for version 1.x alone.